config/locales/codelists/2_03/iati.en.yml

```diff
@@ -91,6 +91,258 @@ en:
+      LB: Lebanon
+      LC: Saint Lucia
+      LI: Liechtenstein
+      LK: Sri Lanka
+      LR: Liberia
+      LS: Lesotho
+      LT: Lithuania
+      LU: Luxembourg
+      LV: Latvia
+      LY: Libya
+      MA: Morocco
+      MC: Monaco
+      MD: Moldova (the Republic of)
+      ME: Montenegro
+      MF: Saint Martin (French part)
+      MG: Madagascar
+      MH: Marshall Islands (the)
+      MK: North Macedonia
+      ML: Mali
+      MM: Myanmar
+      MN: Mongolia
+      MO: Macao
+      MP: Northern Mariana Islands (the)
+      MQ: Martinique
+      MR: Mauritania
+      MS: Montserrat
+      MT: Malta
+      MU: Mauritius
+      MV: Maldives
+      MW: Malawi
+      MX: Mexico
+      MY: Malaysia
+      MZ: Mozambique
+      NA: Namibia
+      NC: New Caledonia
+      NE: Niger (the)
+      NF: Norfolk Island
+      NG: Nigeria
+      NI: Nicaragua
+      NL: Netherlands (the)
+      NP: Nepal
+      NR: Nauru
+      NU: Niue
+      NZ: New Zealand
+      OM: Oman
+      PA: Panama
+      PE: Peru
+      PF: French Polynesia
+      PG: Papua New Guinea
+      PH: Philippines (the)
+      PK: Pakistan
+      PL: Poland
+      PM: Saint Pierre and Miquelon
+      PN: Pitcairn
+      PR: Puerto Rico
+      PS: Palestine, State of
+      PT: Portugal
+      PW: Palau
+      PY: Paraguay
+      QA: Qatar
+      RE: Réunion
+      RO: Romania
+      RS: Serbia
+      RU: Russian Federation (the)
+      RW: Rwanda
+      SA: Saudi Arabia
+      SB: Solomon Islands
+      SC: Seychelles
+      SD: Sudan (the)
+      SE: Sweden
+      SG: Singapore
+      SH: Saint Helena, Ascension and Tristan da Cunha
+      SI: Slovenia
+      SJ: Svalbard and Jan Mayen
+      SK: Slovakia
+      SL: Sierra Leone
+      SM: San Marino
+      SN: Senegal
+      SO: Somalia
+      SR: Suriname
+      SS: South Sudan
+      ST: Sao Tome and Principe
+      SV: El Salvador
+      SX: Sint Maarten (Dutch part)
+      SY: Syrian Arab Republic
+      SZ: Eswatini
+      TC: Turks and Caicos Islands (the)
+      TD: Chad
+      TF: French Southern Territories (the)
+      TG: Togo
+      TH: Thailand
+      TJ: Tajikistan
+      TK: Tokelau
+      TL: Timor-Leste
+      TM: Turkmenistan
+      TN: Tunisia
+      TO: Tonga
+      TR: Turkey
+      TT: Trinidad and Tobago
+      TV: Tuvalu
+      TW: Taiwan (Province of China)
+      TZ: Tanzania, United Republic of
+      UA: Ukraine
+      UG: Uganda
+      UM: United States Minor Outlying Islands (the)
+      US: United States of America (the)
+      UY: Uruguay
+      UZ: Uzbekistan
+      VA: Holy See (the)
+      VC: Saint Vincent and the Grenadines
+      VE: Venezuela (Bolivarian Republic of)
+      VG: Virgin Islands (British)
+      VI: Virgin Islands (U.S.)
+      VN: Viet Nam
+      VU: Vanuatu
+      WF: Wallis and Futuna
+      WS: Samoa
+      XK: Kosovo
+      YE: Yemen
+      YT: Mayotte
+      ZA: South Africa
+      ZM: Zambia
+      ZW: Zimbabwe
+      'false': Norway
```

Comment:
Eeek! YAML recognises "NO" as "false", so to have "NO" in this list you need to encapsulate it in quotes, i.e. `'NO'`

Reply:
Waaaaa?! Thank you! Lesson learnt!